src/focus.service.ts

```diff
@@ -509,6 +509,11 @@ export class FocusService {
   private isFocusable(el: HTMLElement) {
     const role = el.getAttribute('role');
     const tabIndex = el.tabIndex;
+    const visible = window.getComputedStyle(el).display !== 'none';
```

Comment:
Ok. Btw I just tested navigation on xbox and navigating around seems fast enough, so I think the performance hit here is ok.